changes/cmd/changetool/change.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/aggagen/changes"
+	"log"
+	"os"
+	"strconv"
+)
+
+var changeParams = struct {
+	module string
+}{}
+
+var addFlags *flag.FlagSet
+var lsFlags *flag.FlagSet
+
+func init() {
+	addFlags = flag.NewFlagSet("add", flag.ExitOnError)
+	addFlags.StringVar(&changeParams.module, "module", "", "creates a change for the specified module")
+
+	lsFlags = flag.NewFlagSet("ls", flag.ExitOnError)
+	lsFlags.StringVar(&changeParams.module, "module", "", "filters changes by module")
+}
+
+func changeSubcmd(args []string) {
+	if len(args) == 0 {
+		usage()
+	}
+
+	changesPath, err := changes.GetChangesPath()
+	if err != nil {
+		log.Fatalf("Failed to load .changes directory: %v", err)
+	}
+
+	metadata, err := changes.LoadMetadata(changesPath)
+	if err != nil {
+		log.Fatalf("Failed to load .changes directory: %v", err)
+	}
+
+	switch args[0] {
+	case "add", "new":
+		addFlags.Parse(args[1:])
+		addCmd(metadata, changeParams.module)
+	case "ls", "list":
+		lsFlags.Parse(args[1:])
+		lsCmd(metadata, changeParams.module)
+	case "modify", "edit":
```

Comment:
This likely didn't get discussed originally, but we should probably have a non visual mode that would allow our release automation system to feed in the change descriptions for service modules that are being updated. Maybe just pure CLI flags to feed in the module, type, and description.